### PR TITLE
Migration to Hapi v11.x.x and fix deleted method, server.after

### PR DESCRIPTION
**Pull request by njl07**
Hello,

I wrote the fix to resolve the issue #36. Can you check it and merge ASAP.

Thanks in advance.

Best regards

### CODE REVIEW

**Comment by jedireza** [comment]
Thanks for creating the PR. I left a note about changing the module version in the package. I'll do that when I cut the release. Also these changes will require a full version bump.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-mongo-models",
-  "version": "2.2.2",
+  "version": "2.3.0",
```
Comment (jedireza) [inline package.json]: Please remove this version change.

**Comment by njl07** [comment]
Update done.

Note:
I think tests should be launched in `Travis` with latest version of `Node.js` too.
It doesn't have consequence but it better to have the latest version too.

Best regards.

**Comment by jedireza** [comment]
Ok great! Let's squash the commits into one that reads `support for hapi 11.x`.

If you need help with squashing commits see:

 - https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit
 - http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Squashing-Commits

**Comment by jedireza** [comment]
I'm doing some house keeping and updated some other deps too. Closing this in favor of #38.

Thanks again for creating a PR and getting involved.

**Pull request closed without merging**